bin/yang_get_stats.py

```diff
@@ -91,15 +91,20 @@ def __init__(self, args: argparse.Namespace, config: ConfigParser = create_confi
         self.draft_path_nostrict = os.path.join(self.ietf_directory, 'draft-with-YANG-no-strict')
         self.draft_path_diff = os.path.join(self.ietf_directory, 'draft-with-YANG-diff')
         self.stats_path = os.path.join(self.web_private, 'stats')
+        self.files: list[str] = []
+        self.remove_old_html_file_paths: list[str] = []
 
     def start_process(self):
         all_files = self._list_of_files_in_dir(self.backup_directory, 'html')
```

Comment:
I would move this `_list_of_files_in_dir` to some utility.py script, I have a feeling that this is used in multiple scripts.
We often needs to get list of all the `.yang` or `.txt` files in certain directory.
It should have several arguments
- `recursive` - find files recursivelly also in sub-dirs
- `extension` - file extension to find (this could be also as a list)

Identify other scripts where we are doing this file listing/searching.
Make this change either in this PR OR create separate github issue for this please.